test/check-compose-exit-codes.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+shift $(($OPTIND - 1))
+TARGETS="$@"
+
+# check for container exit codes other than 0
+EXIT_CODE=0
+ALL_TESTS=$(docker-compose ps -q ${TARGETS})
+for test in $ALL_TESTS; do
+    docker inspect -f "{{ .State.ExitCode }}" $test | grep -q ^0;
+    if [ $? -ne 0 ]; then 
+        EXIT_CODE=$?;
+        break
+    fi
+done
+
+exit $EXIT_CODE
```

Comment:
I don't see where this file is actually used....

Reply:
Gah, indeed, I didn't mean to commit this. It was from a route I started and decided against.